src/docs/product/performance/web-vitals.mdx

```diff
@@ -71,6 +73,49 @@ Some Web Vitals such as FP, FCP, LCP, and TTFB are measured relative to the star
+
+### Samples List
+
+At the center of the **Page Overview**, Web Vital scores and suspect tags are displayed. Clicking a Web Vital score will open a slideout panel containing a variety of **Page Load** samples with good to poor scores. Each sample contains an **Event ID** that can be clicked to open the [Event Detail](/product/sentry-basics/concepts/tracing/event-detail/) page for further investigation. If there is a [Replay](/product/session-replay/) or [Profile](/product/profiling/) associated with the sample page load, links will be included in the associated table columns.
+
+## Performance Score
+
+**Performance Scores** are a number rating from 0 to 100 that summarizes the percieved performance of your web application. Sentry provides a single score for the overall performance of your application, individual scores for each of your pages, and a score for each sample page load of any of your application's pages.
+
+The **Performance Score** is comprised of 5 individual **Web Vital** components. Each **Web Vital** is given a rating between 0 to 100 through a [Log-Normal Distribution](https://www.desmos.com/calculator/o98tbeyt1t). Each **Web Vital** rating is given a weight, which is summed up to create the overall **Performance Score** with a maximum value of 100. The p90 and p50 distribution values of each **Web Vital** can be found in the table below:
+
+| Web Vital                                                       | P90    | P50    | Weight |
+| --------------------------------------------------------------- | ------ | ------ | ------ |
+| [Largest Contentful Paint](#largest-contentful-paint-lcp) (LCP) | 1200ms | 2400ms | 30%    |
+| [First Input Delay](#first-input-delay-fid) (FID)               | 100ms  | 300ms  | 30%    |
+| [Cumulative Layout Shift](#cumulative-layout-shift-cls) (CLS)   | 0.1    | 0.25   | 15%    |
+| [First Contentful Paint](#first-contentful-paint-fcp) (FCP)     | 900ms  | 1600ms | 15%    |
+| [Time To First Byte](#time-to-first-byte-ttfb) (TTFB)           | 200ms  | 400ms  | 10%    |
+
+## Opportunity
+
+**Opportunity** scores are number values associated with each page and are meant to give you a sense of which pages are most valuable to improve. The **Opportunity** score is the maximum possible increase to your application's overall [Performance Score](#performance-score), if you were to raise a page's individual score to 100. If a page in your application already has a score of 100, the **Opportunity** score would be 0, since there is no further potential to increase your applications overall [Performance Score](#performance-score) by improving this page.
```

Comment:
```suggestion
**Opportunity** scores are number values associated with each page and are meant to give you a sense of which pages are most valuable to improve. The Opportunity score is the maximum possible increase to your application's overall [Performance Score](#performance-score), if you were to raise a page's individual score to 100. If a page in your application already has a score of 100, the Opportunity score would be 0, since there is no further potential to increase your application's overall [Performance Score](#performance-score) by improving this page.
```